tests/conftest.py

```diff
@@ -930,13 +965,12 @@ def _make_libp2p_connection(
         public_key=agent_key.public_key,
         default_address_key=agent_key.identifier,
     )
-    if node_key_file is not None:
-        conn_crypto_store = CryptoStore({DEFAULT_LEDGER_LIBP2P_NODE: node_key_file})
-    else:
-        node_key = make_crypto(DEFAULT_LEDGER_LIBP2P_NODE)
-        node_key_path = os.path.join(data_dir, f"{node_key.public_key}.txt")
-        node_key.dump(node_key_path)
-        conn_crypto_store = CryptoStore({node_key.identifier: node_key_path})
+
+    node_key = make_crypto(DEFAULT_LEDGER_LIBP2P_NODE)
+    node_key_path = os.path.join(data_dir, f"{node_key.public_key}.txt")
+    node_key.dump(node_key_path)
+    conn_crypto_store = CryptoStore({node_key.identifier: node_key_path})
```

Comment:
we never passed the node_key_file. In fact, if we would have done that, a local variable references before assignment error would have been thrown because of `node_key`. 

https://github.com/valory-xyz/open-aea/blob/ca78c2925fd5877451456983ae5816b9b538b274/tests/conftest.py#L952